### ISSUE TITLE
Permission or Redirection issues

### ISSUE THREAD

**Issue by michaelvgibson** [MicrosoftDocs/OfficeDocs-SharePoint#129]
Hi all.  I followed the directions above to try and move my SP2016 databases off of old  hardware and onto a virtual server.  For the most part, everything went pretty smoothly.  When restarting the farm, I was unable to get to Central Admin or the actual SP site.  Event log showed permission issues and that SP could not find the Sharepoint_Config database....despite using cliconfig to create the alias.  I had to roll back unfortunately.  I will diagnose the issues further and hopefully be able to recommend some added steps somewhere.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 3659f8a1-257c-e35e-fac9-5f1094217ddc
* Version Independent ID: 8915a0d7-882a-8e62-c0b6-3142dee12f95
* Content: [Move all databases in SharePoint Server](https://docs.microsoft.com/en-us/sharepoint/administration/move-all-databases#feedback)
* Content Source: [SharePoint/SharePointServer/administration/move-all-databases.md](https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/blob/live/SharePoint/SharePointServer/administration/move-all-databases.md)
* Product: **sharepoint-server-itpro**
* GitHub Login: @bentoncity
* Microsoft Alias: **stevhord**

**Comment by bentoncity** [comment]
Thanks @michaelvgibson, definitely let us know what you find. We tested this successfully for SP2016 pre-release but haven't tested it since.

**Comment by michaelvgibson** [comment]
Thanks @bentoncity.  I think I have isolated the problem.  So for some reason, the alias is not redirecting traffic looking for the old SQL server to the new SQL server.  I am trying to diagnose this issue now.

**Comment by michaelvgibson** [comment]
Ok, the issue ended up being related to the 32 bit cliconfig utility.  This was mentioned in the closed comments on issue #17.  They stated that why specify 32 bit when all the components are x64...so i think those instructions were removed.  I would add the x32 part back into the documentation.  Thanks so much.  Mike.

**Comment by kenwith** [comment]
@tseward - regarding the part that was removed in issue 17.. maybe it should be added back as a workaround mentioned above? Thoughts? 

@michaelvgibson - thanks for help on this! Much appreciated to make the docs better!

**Comment by kenwith** [comment]
@michaelvgibson - I was able to get this into the backlog for the SharePoint team. The team will get this prioritized into a sprint plan. Thank you for reporting and making the docs better. Much appreciated. And sorry for the long delay!

@bentoncity - I added this to the backlog so we can prioritize in upcoming sprint. Just FYI so if you see it you know. I added all the notes too so we can find the right team to get this addressed on product team.

I made a note to request the team to update this when the work is complete.
